1-the-basics/style.css

```
@@ -1,27 +1,30 @@
-header, footer{
-    background-color: limegreen;
+aside{
+    font-size: larger;
+}
+button:hover{
+    background-color:dodgerblue;
 }
 form{
-    background-image: url(https://s1.1zoom.me/b5050/192/Lapland_region_Finland_510265_3840x2400.jpg);
+    background-color: darkorchid;
     background-size: cover;
-    background-color: whitesmoke;
 }
-button:hover{
-    background-color: violet;
+footer{
+    background: chocolate;
 }
-aside{
-    font-size: larger;
+header, footer{
+    background-color: limegreen;
 }
 p{
     font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
 }
-#color{
-    color: brown;
+section{
+    background-image: url(https://s1.1zoom.me/b5050/192/Lapland_region_Finland_510265_3840x2400.jpg); 
 }
 .font{
     font-weight: bold;
 }
-footer{
-    background: chocolate;
+#color{
```

It's better to change the id, `color` does not have much sense
```suggestion
#color {
```